appveyor.sh

```diff
@@ -11,9 +11,9 @@ bash generateprojects.sh && git diff --exit-code
 if [[ -z "$APPVEYOR_PULL_REQUEST_NUMBER" ]]
 then
   # Not in a pull request.
-  # Run a full build (no --diff) with tests and coverage, then create and upload coverage report.
-  ./build.sh --coverage
-  ./createcoveragereport.sh --upload --upload_reportname unittests
+  # Run a full build (no --diff) with tests. Don't do any coverage so it
+  # won't clash with coverage generated through Kokoro.
+  ./build.sh
 else
   # We're in a pull request. Don't do any coverage; just
   # build and run tests.
```

Comment:
It's probably worth amending this comment as well

Reply:
Yep, will create a commit for that. And merge on AppVeyor green.